### PR TITLE
correction on quaternion conversion according to the refer.

### PR DESCRIPTION
**Pull request by taegoobot**

### CODE REVIEW

**Comment by ildoonet** [comment]
@taegoobot 

아래 부분이 실행되지 않는 파트이지만, merge후 처리하기로 하고 merge 했습니다. pull request를 날리는 코드는 테스트를 하고 날려주세요. 

        trace = tf.reduce_sum(tf.matrix_diag_part(matrix_rt, axis=-1)) - 1.0
